arrows/ffmpeg/ffmpeg_video_output.h

```diff
@@ -51,11 +51,11 @@ class KWIVER_ALGO_FFMPEG_EXPORT ffmpeg_video_output
   bool good() const override;
 
   void add_image(
-    kwiver::vital::image_container_sptr const& image,
-    kwiver::vital::timestamp const& ts ) override;
+    vital::image_container_sptr const& image,
+    vital::timestamp const& ts ) override;
   void add_image( vital::video_raw_image const& image );
 
-  void add_metadata( kwiver::vital::metadata const& md ) override;
+  void add_metadata( vital::metadata const& md ) override;
```

Comment:
Not saying this is wrong, but I am noticing inconsistent use of kwiver::vital vs vital vs ::kwiver::vital throughout kwiver. Not sure which one is the best.

Reply:
There are legitimate use cases for `::kwiver::` instead of `kwiver::` but I don't think we have that case here.  If I created nested namespaces `blah::kwiver::` and my scope is currently in the `blah` namespace then `kwiver::` becomes ambiguous and you need to use `::kwiver::` to indicate the kwiver namespace in the global namespace.

I would not use `::kwiver::` unless there is a case where it is needed.